src/server/pegasus_server_impl.h

```diff
@@ -325,6 +328,15 @@ class pegasus_server_impl : public pegasus_read_service
         return dsn::rand::next_u64(base_value - gap, base_value + gap);
     }
 
+    // return true if value in range of [0.75, 1.25] * base_value
+    bool check_value_if_nearby(uint64_t base_value, uint64_t check_value)
+    {
+        uint64_t gap = base_value / 4;
+        uint64_t actual_gap =
+            (base_value < check_value) ? check_value - base_value : base_value - check_value;
```

Comment:
std::abs(check_value - base_value);

Reply:
in this case, the following error will be reported:
```
error: call of overloaded ‘abs(uint64_t)’ is ambiguous
```